src/xdgiconloader/xdgiconloader.cpp

```diff
@@ -777,6 +777,7 @@ QSize XdgIconLoaderEngine::actualSize(const QSize &size, QIcon::Mode mode,
     return {0, 0};
 }
 
+#ifndef QTXDG_STATIC_BUILD
```

Comment:
@ilya-fedin How will be `QTXDG_STATIC_BUILD` toggled ?

Reply:
Since upstream cmake files have only SHARED variant, it could be defined in custom cmake files in a project that want to link with libqtxdg statically

Reply:
Making have an STATIC variant is really easy, CMake supports it. Will do it.
Pushing it to this branch, Ok ?

Reply:
Ok